packages/sdk-drop-in/playwright.config.ts

```diff
@@ -0,0 +1,35 @@
+import { defineConfig, devices } from '@playwright/test';
+
+const config = defineConfig({
+  testDir: './tests',
+  timeout: 30_000,
+  expect: {
+    timeout: 20_000,
+  },
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: [['html', { open: 'never' }]],
+  use: {
+    actionTimeout: 0,
+    baseURL: 'http://localhost:5273',
+    trace: 'on-first-retry',
+    headless: true,
+  },
+  webServer: {
+    command: 'yarn dev --port 5273',
```

Comment:
May I ask you to add a comment that `yarn build && yarn preview` is not used here as routing support is needed, which is implemented for `dev` mode?

Reply:
Done, comment added